geode-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisServer.java

```diff
@@ -275,7 +275,8 @@ public GeodeRedisServer(String bindAddress, int port) {
    *
    * @param bindAddress The address to which the server will attempt to bind to
    * @param port The port the server will bind to, will use {@value #DEFAULT_REDIS_SERVER_PORT}
-   *        by default if argument is less than -1. If the port is {@value #RANDOM_PORT_INDICATOR}
+   *        by default if argument is less than {@value #RANDOM_PORT_INDICATOR}. If the port is
```

Comment:
Is it correct that if it is < RANDOM_PORT_INDICATOR it will use the default? I couldn't find the code that made that happen but it seems odd if I say -33456 it will listen on 6379. I think I'd prefer an IllegalArgumentException

Reply:
That is currently what happens. I like what you're suggesting though.